packages/roosterjs-editor-plugins/lib/pluginUtils/DragAndDropHelper.ts

```diff
@@ -67,22 +67,21 @@ export default class DragAndDropHelper<TContext, TInitValue> implements Disposab
      * so that the handler object knows which element it is triggered from.
      * @param onSubmit A callback that will be invoked when event handler in handler object returns true
      * @param handler The event handler object, see DragAndDropHandler interface for more information
+     * @param zoomScale The zoom scale of the editor
+     * @param forceMobile A boolean to force the use of touch controls for the helper
```

Comment:
You should still mention this is used for test code only to help others understand why we have this parameter.

Reply:
It can be used for more than testing